Add a topic that lists all available OOPE modules (#546)
AdobeDocs/commerce-extensibility: src/pages/oope-modules/api-integration.md

curl --request GET \
--url <ADOBE_COMMERCE_API_URL>/rest/V1/order-statuses \
--header 'Authorization: Bearer <admin_token>'
```

**Example response**:

```json
[
  {
    "status": "pending",
    "label": "Pending",
    "state": "new",
    "is_default": true,
    "visible_on_front": false
  },
  {
    "status": "pending_payment",
    "label": "Pending Payment",
    "state": "pending_payment",
    "is_default": true,
    "visible_on_front": false
  },
  {
    "status": "processing",
    "label": "Processing",
    "state": "processing",
    "is_default": true,
    "visible_on_front": true
  },
  {
    "status": "complete",
    "label": "Complete",
    "state": "complete",
    "is_default": true,
    "visible_on_front": true
  },
  {
    "status": "custom_status",
    "label": "My Custom Status",
    "state": null,
    "is_default": false,
    "visible_on_front": false
  }
]
```

## ACL resources

| Resource ID | Title | Description |
|-------------|-------|-------------|
| `Magento_OopeApiIntegrations::oope` | OOPE API Integrations | Parent resource. |
| `Magento_OopeApiIntegrations::order_statuses` | Order Statuses | Required to call the order statuses endpoint. |

## Installation

> This module is available automatically on Adobe Commerce as a Cloud Service. For on-premises and cloud infrastructure projects, install it with Composer:

Reviewer: ```suggestion
This module is included by default in Adobe Commerce as a Cloud Service. For on-premises or cloud infrastructure projects, install it using Composer.
```
